src/table/src/TableRow.js

```diff
@@ -83,6 +81,7 @@ const TableRow = memo(
           onKeyDown={handleKeyDown}
           height={height}
           borderBottom="muted"
```

Comment:
Similar comment about pushing `borderBottom` into the props config too. I _think_ `splitBoxProps` should know how to spit that back out, but please correct me if I'm wrong.